native/src/main/java/io/ballerina/stdlib/graphql/runtime/engine/ArgumentHandler.java

```diff
@@ -516,7 +516,7 @@ private Object[] getArgumentsForMethod() {
         return result;
     }
 
-    private static Type getEffectiveType(IntersectionType intersectionType) {
+    public static Type getEffectiveType(IntersectionType intersectionType) {
```

Comment:
No need to make this public, right?

```suggestion
    static Type getEffectiveType(IntersectionType intersectionType) {
```